tests/__init__.py

```diff
@@ -244,9 +243,8 @@ def test_isSleeping(self):
                 universal_newlines=True,
             )
         with unittest.mock.patch("hddfancontrol.subprocess.check_output") as subprocess_check_output_mock:
-            subprocess_check_output_mock.side_effect = subprocess.CalledProcessError(0, "")
```

Comment:
Same remark as above, and this a test for `isSleeping` so we should be testing that, not `getState()`.

Reply:
Fixed